packages/nimble-components/src/patterns/link/styles.ts

```diff
@@ -0,0 +1,69 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    linkActiveDisabledFontColor,
+    linkActiveFont,
+    linkActiveFontColor,
+    linkActiveProminentFont,
+    linkActiveProminentFontColor,
+    linkDisabledFontColor,
+    linkFont,
+    linkFontColor,
+    linkProminentFont,
+    linkProminentFontColor,
+    linkProminentDisabledFontColor
+} from '../../theme-provider/design-tokens';
+
+export const linkStyles = css`
+    ${display('inline')}
```

Comment:
If the breadcrumb-item is just going to immediately override this then doesn't seem useful as a shared style. This should just be removed from here and moved to the anchor css (similar to breadcrumb)

Reply:
Done